columnflow/production/example.py

```diff
@@ -6,7 +6,6 @@
 
 from columnflow.production import Producer, producer
 from columnflow.production.normalization import normalization_weights
-from columnflow.production.pileup import pu_weight
```

Comment:
I understand why you removed the CMS specific modules from the example producer. It might make sense to create a CMS specific example though, just to make sure this knowlegde doesn't get lost!

Reply:
With #142 opened, I think would anyway drop the example files as we could have all example snippets (also in e.g. a cms-specific flavor) in a central space (the "analysis_template" subdirectory). This is of course up to discussion, but I think it would be favorable to have only a single place for these type of examples.

In this case, we should of course adapt #142 to remove the `example.py`'s.